scripts/arangodb-bash-setup/arangodb-bash-setup.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+set -e
+
+# IMPORTANT: Consult sample.env in apps/api/src/app/config for 
+# environment configuration prior to running this script
+
+# Check if this bash setup script is being run within it's own directory
+VALID_EXECUTE_DIRECTORY="coscrad/scripts/arangodb-bash-setup"
+CURRENT_PATH="$PWD";
+echo $'\nCURRENT_PATH: '$CURRENT_PATH;
+
+
+# Split path on /
+OLDIFS="$IFS";
+IFS='/';
+read -a strarr <<< $CURRENT_PATH;
+IFS="$OLDIFS";
+length=${#strarr[@]};
+p1=${strarr[$(($length-3))]};
+p2=${strarr[$(($length-2))]};
+p3=${strarr[$(($length-1))]};
+current_directory="${p1}/${p2}/${p3}";
+echo $'\nCurrent Directory: '$current_directory;
+
+if  [ $current_directory != $VALID_EXECUTE_DIRECTORY ];
+then
+  echo $'\n >> Please execute this script within its enclosing directory';
+  echo $'\nEXITING SETUP';
+  echo $'\n';
+  exit;
+else
+  echo $'\n Current directory is valid for executing COSCRAD setup script';
+fi
+
+# Register COSCRAD_ENVIRONMENT mode
+if [ $1 ]; then
+  COSCRAD_ENVIRONMENT=$1
+  export COSCRAD_ENVIRONMENT;
+else
+  echo "No COSCRAD Environment mode input argument supplied.  Accepted: development | test | staging | production";
```

Comment:
What is this used for? Realistically, we would probably only ever do this in staging- maybe development. To reiterate what I've said before, we would never, ever, ever, ever dump the dummy data or drop existing collections in production. You might exit with an error if the environment is production.